### ISSUE TITLE
Transition "Issues" to "Topics"

### ISSUE THREAD

**Issue by sdillingham** [SCPR/SCPRv4#383]
- [x] simplify tag page design to remove all the "Issues"-specific UI cruft (Needs comps from design)
- [x] update routes to support both `/issues/<tag-slug>` (deprecated) and `/topics/<tag-slug>`
- [x] have `/issues/` redirect to homepage(?)
- [x] Change "More from this Issue" sidebar heading to "More from this Topic"
- [x] Change "Issues We're Tracking" heading on Vertical pages to "Topics We're Tracking" and remove "See All Issues" link
- [x] Rename "Issues"-related controllers and views to "Topics"

**Comment by Ravenstine** [comment]
@sdillingham Since we are now distinguishing our tags between `Keyword`, `Beat`, and `Series`, are we still going to have our Issues(now Topics) page include all tags?  Or do we want to restrict it to a type(s)? (e.g. keywords and beats)

**Comment by sdillingham** [comment]
TL;DR - let's have our topic pages include all tags.

As part of rolling out this change, We'll redesign these topic pages. To start, we'll remove the list of tags down the left side of the page. A second phase will probably involve creating different templates for each tag type, but we haven't defined specific requirements for that yet.

**Comment by Ravenstine** [comment]
Great, I think I can work on the other criteria of the AC and integrate the new designs later.

**Comment by Ravenstine** [comment]
@sdillingham On staging, I have made the changes that I have check-marked from the AC.  Here's a pull request to see the changes in code:
https://github.com/SCPR/SCPRv4/pull/393

Barring any fixes I might need to make to the tests(or possibly stylesheets), it seems like the only main thing this is waiting on is comps from design for the simplified tag page.

**Comment by Ravenstine** [comment]
Also, the problem of the tag attributes not persisting on save seems to be gone.  I'm guessing it was an issue with the schema not loading properly.  But it seems to no longer be an issue.

**Comment by sdillingham** [comment]
Thanks for the heads up @Ravenstine, I'll check it out on staging.
